servers/src/main/java/tachyon/worker/block/BlockMasterSync.java

```diff
@@ -69,6 +70,10 @@
   private volatile boolean mRunning;
   /** The id of the worker */
   private long mWorkerId;
+  /** The thread pool to remove block */
+  private static final int DEFAULT_BLOCK_REMOVER_POOL_SIZE = 10;
```

Comment:
@fanshiqing yeah, what you suggested makes sense. I'd like to merge this PR then create a format fix jira on this.